utils.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
+package houdini
+
+import (
+	"os"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"syscall"
+
+	"github.com/cloudfoundry-incubator/garden"
+)
+
+func RunCommandAsUser(cmd *exec.Cmd, spec garden.ProcessSpec) error {
+	runAs, err := user.Lookup(spec.User)
+	if err != nil {
+		return err
+	}
+	uid, err := strconv.ParseUint(runAs.Uid, 10, 32)
+	if err != nil {
+		return err
+	}
+	gid, err := strconv.ParseUint(runAs.Gid, 10, 32)
+	if err != nil {
+		return err
+	}
+
+	os.Chown(cmd.Dir, int(uid), int(gid))
```

Comment:
missing `err` check here. also feels a little strange for this to be here; is it actually necessary? shouldn't it be recursive if so?